integration-tests/README.md

```diff
@@ -1,131 +1,224 @@
 
-Start the docker container which encapsulates the flextesa_sandbox:
+To target a specific test within the suite, use the Jest `--testNamePattern=<regex>` parameter or `-t` for short. Specify the "spec name" of the test you want to run. With `npm`, the command looks like this:
 
-`docker run --rm --name flextesa_sandbox --detach -p 20000:20000 -e block_time=1 oxheadalpha/flextesa:latest limabox start`
+`npm run test -- -t "Title of test here"`
 
-Flextesa is the "Flexible Tezos Sandbox" and effectively enables you to run a local copy of the blockchain. Please find [more information about Flextesa here](https://tezos.gitlab.io/flextesa/).
+For example:
 
-3. Run the integration tests
+```
+npm run test -- -t "Originate FA1.2 contract and fetch data from view entrypoints"
+```
 
-*Note: It is no longer necessary to `cd` into the `integration-tests/` directory*
+## Configuration
 
-To run the integration tests, invoke `npm run integration-tests`.
+Refer to the `taquito/integration-tests/config.ts` file for details on test configurations and target networks. Some configurations have default values that can be overridden using environment variables. In some cases, you can use CLI commands to invoke a configuration instead of exporting it.
 
-The integration test suite runs all tests against the current tezos protocol (lima) sandbox, and typically also against the previous and next protocol testnets. See the `scripts` property in the `integration-tests/package.json` file for specific test targets.
+## CLI Options
 
-Note that the first time you run the integration tests, `docker` will download the image in question, so be patient for your prompt to return the first time: *this is expected*.
+If different testnets are configured in the `config.ts` file, you can run tests on a specific testnet using the command-line parameter for a test file:
+
+```
+npm run test:mumbainet "manager-wallet-scenario.spec.ts"
+```
 
-## How to Run the Integration Tests Against a Tezos Testnet
+Or for a specific test within a test file:
 
-These tests exercise many use cases of Taquito against a live Tezos testnet.
+```
+npm run test:mumbainet -- -t "Originate FA1.2 contract and fetch data from view entrypoints"
+```
 
-To run in this environment, you must have:
+## Running Tests Against a Specific RPC URL
 
-- Taquito source code cloned to your local machine
-- Installed a compatible version of node.js (see top-level README.md)
-- Successfully compiled Taquito (see top-level README)
+To run tests against a node that is not preconfigured in Taquito, use:
 
-## Running all tests against all pre-configured testnets
+`export TEZOS_RPC_MUMBAINET='http://localhost:8732'`.
 
-From the `taquito/integration-tests` directory run
+## Using a Secret Key Instead of the Keygen API
+
+By default, the integration tests use an ephemeral key managed by the Keygen API. To use a secret key instead, use the CLI option `<testnet>-secret-key`, like this:
 
```

Comment:
Also worth mentioning that this API is only available internally

Reply:
done